source/_components/sensor.opensky.markdown

```diff
@@ -42,3 +42,17 @@ Both events have two attributes:
 
 - **sensor**: Name of `opensky` sensor that fired the event
 - **callsign**: Callsign of the flight
+
+To receive notifications of the entering flights, add the following lines to your `configuration.yaml`.
+
+```yaml
+automation:
+  - alias: 'Flight entry notification'
+    trigger:
+      platform: event
+      event_type: opensky_entry
+    action:
+      service: notify.ios_YOURIPHONENAME
+      data_template:
+        message : 'Flight entry of {{ trigger.event.data.callsign }} '
```

Comment:
To avoid render of the template, use `{% raw %}{{your_template}}{% endraw %}`.

Reply:
I don’t know anything about the notation you are suggesting. I was already happy that I got it working as-is.

Reply:
@GrumpyMeow If you go to your repo at GitHub.com, go to the branch `GrumpyMeow:patch-1` where the file is modified, and then edited in-browser, the file will be committed and this will update this pull request. Sounds complicated, I know...

Reply:
The website will render it blank.